terraform/terraform_test.go

```diff
@@ -60,6 +61,54 @@ func testModule(t *testing.T, name string) *module.Tree {
 	return mod
 }
 
+// testModuleInline takes a map of path -> config strings and yields a config
+// structure with those files loaded from disk
+func testModuleInline(t *testing.T, config map[string]string) *module.Tree {
+	cfgPath, err := ioutil.TempDir("", "tf-test")
+	if err != nil {
+		t.Errorf("Error creating temporary directory for config: %s", err)
+	}
+	// defer os.RemoveAll(cfgPath)
```

Comment:
can we clean this up, or do the files need to exist later for the `*module.Tree`?